db/migrate/20250625004725_add_raw_pending_column_transaction_id_to_canonical_pending_transaction.rb

```diff
@@ -0,0 +1,7 @@
+class AddRawPendingColumnTransactionIdToCanonicalPendingTransaction < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
+
+  def change
+    add_reference :canonical_pending_transactions, :raw_pending_column_transaction, index: { algorithm: :concurrently, name: "index_canonical_pending_txs_on_rpct_id" }
```

Comment:
- **question**: Should we add a foreign key?
- **question**: Are we expecting this to be `NULL` sometimes?
- **question**: Should this be a unique index?

Reply:
This will often be `NULL`, in the overwhelming amount of cases this will be `null`. Can be made unique. No opinion on making it a foreign key.